src/main/java/edu/cornell/kfs/coa/businessobject/lookup/AccountGlobalSearchLookupableHelperServiceImpl.java

```diff
@@ -35,6 +36,8 @@
 
 public class AccountGlobalSearchLookupableHelperServiceImpl extends KualiAccountLookupableHelperServiceImpl {
     private AccountGlobalSearchDao accountGlobalSearchDao;
+    private IdentityService identityService;
+    private PermissionService permissionService;
```

Comment:
This subclass doesn't seem to be using the permissionService variable, so this variable and the additions or overrides of the related getters/setters can be removed.

Reply:
Given that Nancy is out of office until next week, and that the extra permissionService variable/getter/setter appear to be set up properly, and that we already have follow-up JIRAs to potentially move our various lookups to the new framework anyway, I'm fine with deferring the permissionService cleanup to a future user story.